AsyncDisplayKit/ASCellNode.h

```diff
@@ -75,6 +78,7 @@ typedef NSUInteger ASCellNodeAnimation;
  */
 @property (nonatomic, weak) id<ASCellNodeLayoutDelegate> layoutDelegate;
 
+@property (nonatomic, assign) BOOL shouldObserveVisibility;
```

Comment:
I'm not sure we should provide this API. It is a bit confusing to clients. This is probably fine for now, Not a reason to block the diff, but it would be important to change this before the next production release of the framework (which could be as early as this weekend) if it will be changed.